wander-embedded-sdk/src/components/iframe/wander-iframe.component.ts

```diff
@@ -182,13 +130,18 @@ export class WanderIframe {
     host.id = options.id || WanderIframe.DEFAULT_IFRAME_ID;
 
     const shadow = host.attachShadow({ mode: "open" });
+    const template = document.createElement("template");
 
-    const backdrop = document.createElement("div");
+    template.innerHTML = getWanderIframeTemplateContent();
+
+    shadow.appendChild(template.content);
 
+    const backdrop = document.createElement("div");
```

Comment:
The iframe and backdrop elements should not be created manually. You can just add them inside the template content and get a reference to them, and update them if needed (`src` attribute).